axiom/client.py

```diff
@@ -53,6 +54,12 @@ def __init__(self, token: str, org_id: str = None, url_base: str = CLOUD_URL):
 
         logger = getLogger()
         session = BaseUrlSession(url_base)
+        # set exponential retries
+        retries = Retry(
+            total=3, backoff_factor=0.1, status_forcelist=[500, 502, 503, 504]
```

Comment:
Can you specify how long to wait between retries? We should probably use a Backofen factor of 2 of exponential backoff.

Reply:
updated